index.js

```diff
@@ -117,39 +50,357 @@ map.on('load', function() {
+        map.addLayer({
+            "id": "issues_1260_1",
+            "type": "symbol",
+            "source": {
+                'type': 'vector',
+                "tiles": ["https://cors.5apps.com/?uri=http://osmose.openstreetmap.fr/fr/map/issues/{z}/{x}/{y}.mvt?item=1260&class=1"],
+                "attribution": "Osmose",
+                "minzoom": 12
+            },
+            "source-layer": "issues",
+            "layout": {
+                "icon-image": "{item}"
+            }
+        });
+        map.on('mouseenter', 'issues_1260_1', function(e) {
+            map.getCanvas().style.cursor = 'pointer';
+        });
+
+        map.on('mouseleave', 'issues_1260_1', function() {
+            map.getCanvas().style.cursor = '';
+        });
+
+        map.on('click', 'issues_1260_1', display_info_1260_1)
+        async function display_info_1260_1(e) {
+            map.flyTo({
+                center: e.features[0].geometry.coordinates,
+                zoom: 18
+            });
+            var popup_content = "<b>Le trajet de cette ligne contient des trous </b></br>"
+            popup_content += "Le tracé est sûrement incomplet, ou erroné."
+
+            popup.style.display = 'block';
+            popup.innerHTML = popup_content
+            var item_id = e.features[0]['properties']['item'];
+            try {
+                var osmose_url = "https://cors.5apps.com/?uri=http://osmose.openstreetmap.fr/fr/api/0.2/error/" + e.features[0].properties.issue_id
+                var osmose_response = await fetch(osmose_url);
+                var osmose_data = await osmose_response.json();
+
+                for (elem_id in osmose_data['elems']) {
+                    elem = osmose_data['elems'][elem_id]
+                    if (elem['type'] == 'relation') {
+                        var external_url = 'http://ra.osmsurround.org/analyzeMap?relationId=' + elem['id'];
+                        popup_content += "<br><a target='blank_' href='" + external_url + "'>Analyser cette relation</a>"
+                    }
+                }
+                popup.innerHTML = popup_content
+
+            } catch (err) {
+                console.log("erreur en récupérant les infos d'Osmose : " + err)
+            }
+        };
+
+    }
+
+    // issues_1260_3
+    if (osmose_issues_to_display === '1260_3' || osmose_issues_to_display === 'all') {
+        map.addLayer({
+            "id": "issues_1260_3",
+            "type": "symbol",
+            "source": {
+                'type': 'vector',
+                "tiles": ["https://cors.5apps.com/?uri=http://osmose.openstreetmap.fr/fr/map/issues/{z}/{x}/{y}.mvt?item=1260&class=3"],
+                "attribution": "Osmose",
+                "minzoom": 12
+            },
+            "source-layer": "issues",
+            "layout": {
+                "icon-image": "{item}"
+            }
+        });
+        map.on('mouseenter', 'issues_1260_3', function(e) {
```

Comment:
Pour ces petites fonctions là, je pense que ça vaudrait le coup de les générer dans une boucle for sur chaque type d'erreur. Qu'est-ce que tu en dis ?
C'est déjà a moitié fait pour les erreurs Omsose génériques, ça pourrait se faire en bouclant sur chaque layer affiché).

Reply:
J'ai hésité à le faire, mais j'ai préféré les garder séparées:
* quand on aura des erreurs Osmose corrigeable directement dans Bifidus, avec bcp de code spécifique pour chaque type d'erreur, on voudra peut-être séparer chaque type d'erreur dans son fichier à part, avec toute la logique qui va bien
* je n'ai pas envie de créer ces fonctions pour chaque layer affiché (il y a bcp de layers affichés, les routes, les rivières, etc), et je préfère aussi éviter de les créer pour des layers d'erreurs Osmose qui ne sont pas affichés, donc ça ajoute de la complexité pour un gain assez limité il me semble

Reply:
j'ai encapsulé ça dans une fonction pour réduire la duplication de code.